nfsn-cron.sh

```diff
@@ -20,8 +11,10 @@ git submodule update --remote
 cd letsencrypt.sh
 
 echo " + Checking certificate expiration date..."
-declare -a checks=( $(find certs -name cert.pem -type l -exec openssl x509 -checkend 2592000 -in {} \;) )
-if contains "Certificate will expire" "${checks[@]}"; then
+if find certs -name cert.pem -type l \
+	-exec openssl x509 -checkend 2592000 -in {} \; |
+	grep -F "Certificate will expire" >/dev/null
```

Comment:
Done. I typically avoid `-q` since it’s not so portable. In this case we know NFSN has grep with `-q` support.